src/components/file-uploader/file-uploader.style.js

```diff
@@ -0,0 +1,42 @@
+export const style = {
+  root: { 
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+    alignItems: 'center',
+    minHeight: '100px',
+    border: 'dashed',
+    borderColor: 'primary.200',
+    borderRadius: '4px',
+    overflow: 'auto',
+  },
```

Comment:
don't duplicate styles
you can do
sx={ [styles.root, isDrag && styles.rootDrag] }
styles.rootDrag will override needed root styles

Reply:
fixed